### ISSUE TITLE
[Modelo]

### ISSUE THREAD

**Issue by benoit-dd** [osm-fr/attributions#516]
### Ce signalement concerne…

cartes interactives avec surcouches d'informations locales à destination des professionnels de l'immobilier, notaires...

https://www.modelo.fr/insight (anciennement cityscan: https://www.cityscan.fr/valorisation-de-bien/ )

filiale de https://www.septeo.com/fr

exemple d'utilisation: https://www.interconstruction.fr/ile-de-france/92/bagneux/virtuo/

### Présence de l'attribution

❌ Pas d'attribution

### Prise de contact

❌ Je ne l'ai pas contacté

### Informations supplémentaires

- Responsable contacté le : 
  - Responsable contacté par (courriel, Twitter…) : [contact@modelo.fr](mailto:contact@modelo.fr)
- Lié au signalement : #

**Comment by NathanBnm** [comment]
Avez-vous contacté l'éditeur du site ? La description de votre demande est contradictoire.

**Comment by benoit-dd** [comment]
non, pas contacté, j'ai juste indiqué le mail de contact utilisable

**Comment by NathanBnm** [comment]
J'ai reçu une réponse aujourd'hui ça a été corrigé. Ils m'ont dit que c'était un oubli suite à une nouvelle version du widget et qu'ils avaient fait le nécessaire partout.

**Comment by NathanBnm** [comment]
C'est corrigé @Jean-BaptisteC
